shaders/color_write_enable/triangle_separate_channels.frag

```diff
@@ -0,0 +1,32 @@
+#version 320 es
+/* Copyright (c) 2023, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+precision mediump float;
+
+layout(location = 0) in vec3 in_color;
+
+layout(location = 0) out vec4 out_color_r;
+layout(location = 1) out vec4 out_color_g;
+layout(location = 2) out vec4 out_color_b;
+
+void main()
+{
+       out_color_r = vec4(in_color, 1.0);
```

Comment:
Maybe a short comment that each of the three outputs is specified to use just one component (via the `blend_attachment` in `ColorWriteEnables::prepare_pipelines`)

Reply:
Done.